src/Chainweb/Pact/Backend/Utils.hs

```diff
@@ -166,7 +166,7 @@ withSavepoint name action = mask $ \resetMask -> do
         liftIO $ evaluate r
     throwErr s = internalError $ "withSavepoint (" <> toText name <> "): " <> s
     handlers = [ Handler $ \(e :: PactException) -> throwErr (sshow e)
-               , Handler $ \(e :: AsyncException) -> throwM e
+               , Handler $ \(e :: SomeAsyncException) -> throwM e
```

Comment:
Oh, good catch. Is this change safe to apply or can it break anything? I guess it should be safe, but why may want quickly grep over all use sites.

Reply:
It's used only in two places and it seems fine. Also checked the respective functions in `Chainweb.Utils`, which have correct behavior.

Reply:
This should be safe I think, any place which is fine with `ThreadKilled` should also be fine with `AsyncCancelled`.